railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow `bin/rails test` to take absolute paths as arguments
```

Comment:
Hi, sorry didn't see the comment in time @lugray, else I'd have updated it. Thanks @kaspth!